### PR TITLE
Documentation for init controller image related changes

### PR DESCRIPTION
**Pull request by kewegner**
Initial documentation changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,47 @@ This can be specified in several ways (note --verbose and -v are interchangeable
...
+- DOCKER_PASSWORD
+
+This will build the init-controller image locally and push it to the corresponding DOCKER_ORG/DOCKER_USERNAME/init-controller:{TRAVIS_TAG} docker repository location where it will also be tagged as 'latest'.
+
+## Testing the controller with the Appsody CLI
+The Appsody CLI makefile specifies a particular version of the appsody-controller.  Which will cause a particular version of init-controller to be used.
+
+If a tester needs to test a different version of the controller with the CLI, there are two environment variables they can exported to test with a different version:
+
+APPSODY_CONTROLLER_VERSION will allow the tester to specify a different version of the appsody-controller, for instance 0.3.0 vs 0.3.1
```
Comment (chilanti) [inline README.md]: Add a colon after 0.3.1:

File: README.md [hunk 2]
```diff
@@ -65,6 +65,47 @@ This can be specified in several ways (note --verbose and -v are interchangeable
...
+
+`export APPSODY_CONTROLLER_VERSION=0.3.0`
+
+APPSODY_CONTROLLER_IMAGE will allow the tester to specify a different image to use, this is very useful during development.
+For instance a developer could create a new image for the controller at docker.io/{org}/init-controller:{tag} and then specify this as the image to use when invoking the CLI.
```
Comment (chilanti) [inline README.md]: Colon instead of full stop after `CLI`

File: README.md [hunk 3]
```diff
@@ -65,6 +65,47 @@ This can be specified in several ways (note --verbose and -v are interchangeable
...
 >--v=true   Debug logging is on  
 
+--version returns the current version
+
+## The docker appsody/init-controller:{travis_tag} image
+
+This image is built as part of the release/deploy process in travis.
+This image contains a copy of the appsody-controller binary placed at the `/` directory of the image.
+
+In addition there is a CMD in the Dockerfile build which will cause the /appsody-controller binary to be copied to the /.appsody directory, which is a volume known to the Appsody CLI.
```
[the remark this answers is not in the document]
Comment (kewegner) [reply README.md]: changed

File: README.md (hunk 1 above)
Comment (kewegner) [reply README.md]: changed

**Pull request merged**